components/performance_counters/io/src/io_counters.cpp

```diff
@@ -28,6 +28,9 @@
 #include <fstream>
 #include <iterator>
 #include <string>
+#if defined(HPX_HAVE_UNISTD_H)
+#include <unistd.h>
+#endif
```

Comment:
Just curious, on which configuration was this not available?

Reply:
It failed with Clang CUDA